### ISSUE TITLE
found 4 moderate severity vulnerabilities

### ISSUE THREAD

**Issue by rubengvillar** [nodejs/node-gyp#1514]
```
                       === npm audit security report ===


                                 Manual Review
             Some vulnerabilities require your attention to resolve

          Visit https://go.npm.me/audit-guide for additional guidance

  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   node-sass [dev]

  Path            node-sass > node-gyp > request > hawk > boom > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   node-sass [dev]

  Path            node-sass > node-gyp > request > hawk > cryptiles > boom >
                  hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   node-sass [dev]

  Path            node-sass > node-gyp > request > hawk > hoek

  More info       https://nodesecurity.io/advisories/566


  Moderate        Prototype pollution

  Package         hoek

  Patched in      > 4.2.0 < 5.0.0 || >= 5.0.3

  Dependency of   node-sass [dev]

  Path            node-sass > node-gyp > request > hawk > sntp > hoek

  More info       https://nodesecurity.io/advisories/566
```

**Comment by brodybits** [comment]
Should be solved by bumping request to 2.87.0, which is in progress ref: #1492

**Comment by maclover7** [comment]
Solved by #1521 (now out in `v3.8.0`)
